### PR TITLE
feat(i18n): Use "J'aime" & "Aimés"

### PR DESCRIPTION
**Pull request by KaKi87**
As per #1083.

### CODE REVIEW

**Review by florian-lefebvre — APPROVED** [review]

File: locales/fr-FR.json [hunk 1]
```diff
@@ -49,8 +49,8 @@
     "confirm": "Confirmer",
     "edit": "Éditer",
     "enter_app": "Entrer dans l'application",
-    "favourite": "Ajouter aux messages aimés",
-    "favourited": "Ajouté aux messages aimés",
+    "favourite": "J'aime",
+    "favourited": "J'aime",
```
Comment (edimitchel) [inline locales/fr-FR.json:53]:
```suggestion
    "favourited": "Aimé",
```

File: locales/fr-FR.json [hunk 2]
```diff
@@ -164,7 +164,7 @@
     "built_at": "Dernière compilation {0}",
     "conversations": "Conversations",
     "explore": "Explorer",
-    "favourites": "Favoris",
+    "favourites": "Aimés",
```
Comment (edimitchel) [inline locales/fr-FR.json:167]: I'm not fan of **Aimés** in the sidenav menu.. I like **Favoris**.

File: locales/fr-FR.json (hunk 1 above)
Comment (KaKi87) [reply locales/fr-FR.json:53]: Yet previous choices were made accordingly to popular social media practises, and that one's as much relevant.

File: locales/fr-FR.json (hunk 2 above)
Comment (KaKi87) [reply locales/fr-FR.json:167]: Yet previous choices were made accordingly to popular social media practises, and that one's as much relevant.

File: locales/fr-FR.json (hunk 1 above)
Comment (KaKi87) [reply locales/fr-FR.json:53]: I didn't check but I assumed the difference between `favourite` and `favourited` is whether the *like* button is active or not, right ? If yes, then using "J'aime" for both situations is accurate in French, isn't it ?

File: locales/fr-FR.json (hunk 2 above)
Comment (edimitchel) [reply locales/fr-FR.json:167]: I'm ok to leave Favoris, WDYT?

Comment (KaKi87) [reply locales/fr-FR.json]: Alright.

File: locales/fr-FR.json (hunk 1 above)
Comment (edimitchel) [reply locales/fr-FR.json:53]: I understand but, actually, I think in the case it's when you like the post, you can change the form of the sentence to give the current status.
THe action is "J'aime", and the result is "Aimé", or "Vous aimez"

Comment (KaKi87) [reply locales/fr-FR.json]: Alright.

**Review by edimitchel — APPROVED** [review]

**Pull request merged**